test/Microsoft.AspNetCore.Mvc.ApiExplorer.Test/DefaultApiDescriptionProviderTest.cs

```diff
@@ -409,9 +410,181 @@ public void GetApiDescription_DoesNotPopulatesResponseInformation_WhenUnknown(st
 
             // Assert
             var description = Assert.Single(descriptions);
-            Assert.Null(description.ResponseType);
-            Assert.Null(description.ResponseModelMetadata);
-            Assert.Empty(description.SupportedResponseFormats);
+            Assert.Empty(description.SupportedResponseTypes);
+        }
+
+        public static TheoryData<Type, string, List<FilterDescriptor>>
```

Comment:
You could do
`public static TheoryData ReturnsActionResultWithProducesAndProducesContentTypeData` to avoid having to split this